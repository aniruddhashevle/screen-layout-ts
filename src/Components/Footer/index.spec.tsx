// #region Global Imports
import * as React from "react";
import { renderWithProviders } from "@Test/Helpers/render";
// #endregion Global Imports

// #region Local Imports
import { Footer } from "@Components";
// #endregion Local Imports

describe("Components", () => {
    describe("Footer", () => {

        it("should match snapshot", () => {
            const wrapper = renderWithProviders(<Footer />);

            expect(wrapper).toMatchSnapshot();
        });
    });
});