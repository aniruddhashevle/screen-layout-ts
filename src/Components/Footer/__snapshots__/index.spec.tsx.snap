// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Footer should match snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#fce0a7",
        "height": 250,
        "justifyContent": "center",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={false}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#f8d332",
        "borderRadius": 25,
        "borderWidth": 1,
        "opacity": 1,
        "paddingBottom": 10,
        "paddingLeft": 35,
        "paddingRight": 35,
        "paddingTop": 10,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontSize": 18,
          },
        ]
      }
    >
      Click here!
    </Text>
  </View>
</View>
`;
